devbox_test.go

```diff
@@ -12,6 +12,7 @@ import (
 )
 
 func TestDevbox(t *testing.T) {
+	os.Setenv("TMPDIR", "/tmp")
```

Comment:
If you use [`t.Setenv`](https://pkg.go.dev/testing#T.Setenv) it'll automatically restore the original value and guard against changing the value in parallel tests.

It might also be worth checking that `/tmp` exists before using it so we can provide a helpful error.